src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx

```diff
@@ -235,24 +226,36 @@ const ShortCircuitAnalysisResultTable: FunctionComponent<
         [theme.selectedRow.background]
     );
 
-    const getCurrent = (
-        faultResult: SCAResultFault | SCAResultFaultFeederResult
-    ) => {
+    const getCurrent = (x: SCAFaultResult | SCAFeederResult) => {
         let current = NaN;
-        if (analysisType === ShortcircuitAnalysisType.ALL_BUSES) {
-            current = faultResult.current;
-        } else if (analysisType === ShortcircuitAnalysisType.ONE_BUS) {
-            current = faultResult.positiveMagnitude;
+        if (analysisType === ShortCircuitAnalysisType.ALL_BUSES) {
+            current = x.current;
+        } else if (analysisType === ShortCircuitAnalysisType.ONE_BUS) {
+            current = x.positiveMagnitude;
         }
         return current;
     };
 
-    const flattenResult = (shortCircuitAnalysisResult: SCAResultFault[]) => {
+    // When we filter / sort the 'current' column in one bus, it's actually the 'fortescueCurrent.positiveMagnitude' field in the back-end
+    const fromColumnToField = useCallback(
+        (column: string) => {
+            if (
+                analysisType === ShortCircuitAnalysisType.ONE_BUS &&
+                column === 'current'
```

Comment:
Why not rename this column whit the correct name as in the back and remove this transcription method which is useless for all other columns which have the correct name ?